themes/custom_theme/valencia-theme/src/css/_custom.scss

```diff
@@ -0,0 +1,565 @@
+/* inject:imports */
+
+
+/* endinject */
+
+// Google Fonts
+@import url('https://fonts.googleapis.com/css?family=Roboto+Slab:400,700|Roboto:300,400,500,700');
+
+/* =============================================================
+>>> TABLE OF CONTENTS:
+================================================================
+# Sass Variables
+# Generic styles
+# Page Styling
+# Navigation
+# Typography
+# Buttons and links
+# Forms
+# Cards styling
+# Tables
+# Sections Styling
+# Footer
+# Responsive
+================================================================ */
+
+// /* ==========================================================
+//       Sass Variables
+// ============================================================= */
+
+/* Classes*/
+$theme: av_theme; // Class for overriding if neccesary
+
+/* Colors */
+$font-h-color: #3C4858;
+$font-p-color: #555;
+$accent-color: #7DBE0D;
+$secondary-color: #0AACCC;
+$tertiary-color: #E8990C;
+$bg-color: #e5e5e5;
+
+/* Fonts */
+$font-sans: Roboto, Helvetica, Arial, sans-serif;
+$font-serif: "Roboto Slab", "Palatino Linotype",Georgia, serif;
+
+/* Media Quieries */
+$screen-xs: "screen and (min-width: 480px)";
+$screen-sm: "screen and (min-width: 768px)";
+$screen-md: "screen and (min-width: 992px)";
+$screen-lg: "screen and (min-width: 1200px)";
```

Comment:
I think we already have screen sizes defined in Liferay.  We should not need to recreate new ones.